src/components/Dashboard/Content/index.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import DashboardContent, { dashboardDataTestIds } from '.';
+
+describe('DashboardContent', () => {
+  const surveys = [
+    {
+      id: '1',
+      resourceType: 'survey',
+      coverImageUrl: 'https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_',
+      title: 'Working from home Check-In',
+      description: 'We would like to know how you feel about our work from home.',
+    },
+  ];
+
+  it('renders DashboardContent and its components', () => {
+    render(
+      <DashboardContent
+        surveys={surveys}
+        currentPosition={0}
+        shouldShowShimmer={false}
+        onNextSurvey={() => jest.fn()}
+        onIndicatorTapped={() => jest.fn()}
+      />
+    );
+
+    const dashboardContent = screen.getByTestId(dashboardDataTestIds.content);
+
+    expect(dashboardContent).toBeVisible();
+    expect(dashboardContent).toHaveTextContent(surveys[0].title);
+    expect(dashboardContent).toHaveTextContent(surveys[0].description);
+  });
+
+  it('does NOT render the DashboardContent components', () => {
```

Comment:
sorry I missed it previously, currently we have only two test blocks:

```
it('renders Dashboard and its contents', () => {
it('does NOT render the DashboardContent components', () => {
```

could we provide a bit more context here?

```js
describe('when shouldShowShimmer is true', ()  => {
  it('renders Dashboard and its contents', () => {
  
describe('when shouldShowShimmer is false', ()  => {
  it('does NOT render Dashboard and its contents', () => {
```